rlinf/workers/actor/megatron_actor_worker.py

```diff
@@ -145,8 +145,12 @@ def __init__(
 
         # Reward configurations
         if not self.cfg.reward.use_reward_model:
-            assert self.cfg.reward.reward_type == "math", "only support math"
-            self.reward_fn = math_verify_call
+            if self.cfg.reward.reward_type == "math":
+                self.reward_fn = math_verify_call
+            elif self.cfg.reward.reward_type == "online_coding":
+                self.reward_fn = None
```

Comment:
using `if/else` here is not elegant, is there a better way? for example, directly passing in the reward function and assigning it to `self.reward_fn`?